conan/internal/util/__init__.py

```diff
@@ -0,0 +1,30 @@
+import math
+import multiprocessing
+
+from conan.internal.util.files import load
+
+
+def cpu_count():
```

Comment:
Nothing has changed in the ``cpu_count()`` function, just a pure refactor/move to this location to make it commonly available, as it was previously private to the ``tools.build`` scope.